resources/assets/js/trainingticket.js

```diff
@@ -83,6 +83,7 @@ const endInput = document.getElementById("end");
 [startInput, endInput].forEach((input) => {
   input.addEventListener("change", () => {
     setTimeout(() => {
+      timeRectifyFormat([startInput, endInput]);
```

Comment:
I'm not sure of the cleanest way to do this, but I think making the duration be "rectified" as well would be clean. Either that or if we always want the duration to be set automatically, we could just gray out that field on the form (but I can see how that could cause issues for edge cases - like a session for some reason goes for more than 24 hours or goes after midnight or goes through a daylight savings time change)